ui/desktop/app/styles/app.scss

```diff
@@ -541,6 +541,25 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
+// Proxy URL
+.proxy-url-container {
```

Comment:
I'm somewhat not a fan of the ssh command or address/port jumping depending on the size of the content of the dropdown, would it be better to have the dropdown be a consistent width?

Reply:
How does this look? I am rending double proxy urls at the moment just to test some style changes.
<img width="795" alt="Screen Shot 2023-08-29 at 10 24 05 AM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/265b83e9-816b-4e3c-8416-abb8dd15f442">
<img width="795" alt="Screen Shot 2023-08-29 at 10 24 12 AM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/c07bc3c5-af23-4561-b5fa-812400564b35">

Reply:
I personally like that more, not sure if design or anyone else has preferences